### PR TITLE
Fixing content-length issue

### PR DESCRIPTION
**Pull request by sinewave440hz**
### Problem
When initialising a session is used the Conviva integration does not inform Conviva of content length. This was previously by design but a reasonable use case for this has become apparent.

### Solution

Ensure that content length is updated regardless of whether or not the session has started, provided that content length is not already set. (This is done by making some minor changes to duration handling in the `ContentMetadataBuilder.swift`)

### Notes

Also updated the example podfile to use the cocoapods CDN.

### Checklist
- [*] I added an update to `CHANGELOG.md` file
- [*] I ran all the tests in the project and they succeed

### CODE REVIEW

File: BitmovinConvivaAnalytics.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'BitmovinConvivaAnalytics'
-  s.version          = '2.0.1'
+  s.version          = '2.0.2'
```
Comment (stonko1994) [inline BitmovinConvivaAnalytics.podspec:3]: Usually, we do set the version when we do the final release. This happens when on develop / a release branch. Please don't update the version in this PR 🙂

File: BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift [hunk 2]
```diff
@@ -69,12 +69,16 @@ class ContentMetadataBuilder {
             if let type = streamType {
                 contentMetadata.streamType = type
             }
-            if let duration = self.duration {
+            if let duration = self.duration, self.duration ?? 0 > 0 {
```
Comment (stonko1994) [inline BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift:72]: You can use the unwrapped value `duration` in follow up conditions 🙂 

```suggestion
            if let duration = self.duration, duration > 0 {
```

File: BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift [hunk 3]
```diff
@@ -351,7 +351,7 @@ public final class ConvivaAnalytics: NSObject {
             self.logger.debugLog(
                 message: "[ ConvivaAnalytics ] false stalling, not registering to Conviva"
             )
-            return;
+            return
```
Comment (stonko1994) [inline BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift:354]: thx 👍 😄

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [2.0.2]
```
Comment (stonko1994) [inline CHANGELOG.md:8]: As the version will be set when we do the final release please change this to `[Unreleased]` for now.

```suggestion
## [Unreleased]
```

File: Example/Podfile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 source 'https://github.com/bitmovin/cocoapod-specs.git'
-source 'https://github.com/CocoaPods/Specs.git'
+source 'https://cdn.cocoapods.org/'
```
Comment (stonko1994) [inline Example/Podfile:2]: thx 🙌 👍

**Review by stonko1994 — COMMENTED** [review]
Would it be possible to cover this changes in unit tests? 🤔

File: BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift (hunk 2 above)
Comment (sinewave440hz) [reply BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift:72]: Much better, thanks!

**Review by stonko1994 — APPROVED** [review]

**Review by luckygoyal-bitmovin — APPROVED** [review]
Changes LGTM

**Comment by sinewave440hz** [comment]
> Would it be possible to cover this changes in unit tests? 🤔

I need to address this. I had a quick go at adding, but I had trouble getting a test failure with the original failing code. I need to spend a few minutes looking at how to do that...happy to discuss in slack...

**Pull request merged**